docs/build.md

```diff
@@ -208,3 +208,212 @@ truffle(zen)> web3.eth.getChainId()
+ ```
+
+Hardhat provides a list of available tasks that you can run. In your project folder, run the command:
+
+```
+npx hardhat
+```
+You will see the following:
+
+```
+$ npx hardhat
+Hardhat version 2.12.0
+
+
+Usage: hardhat [GLOBAL OPTIONS] <TASK> [TASK OPTIONS]
+
+
+GLOBAL OPTIONS:
+
+
+  --config              A Hardhat config file.
+  --emoji               Use emoji in messages.
+  --help                Shows this message, or a task's help if its name is provided
+  --max-memory          The maximum amount of memory that Hardhat can use.
+  --network             The network to connect to.
+  --show-stack-traces   Show stack traces.
+  --tsconfig            A TypeScript config file.
+  --verbose             Enables Hardhat verbose logging
+  --version             Shows hardhat's version.
+
+
+
+
+AVAILABLE TASKS:
+
+
+  check                 Check whatever you need
+  clean                 Clears the cache and deletes all artifacts
+  compile               Compiles the entire project, building all artifacts
+  console               Opens a hardhat console
+  coverage              Generates a code coverage report for tests
+  flatten               Flattens and prints contracts and their dependencies
+  help                  Prints this message
+  node                  Starts a JSON-RPC server on top of Hardhat Network
+  run                   Runs a user-defined script after compiling the project
+  test                  Runs mocha tests
+  typechain             Generate Typechain typings for compiled contracts
+  verify                Verifies contract on Etherscan
+
+To get help for a specific task run: npx hardhat help [task]
+```
+
+
+Hardhat provides a list of all the tasks you can perform. However, you will be performing the run, test, and compile tasks.
+
+
+### Add the ZEN Network
+In order to deploy your smart contract, you will need a network. In the configuration file, **hardhat.config.ts**, replace the content with the following:
+
+**Note:** In the account field, replace “word1 … word12” with a valid seed phrase.
+
+```
+import { HardhatUserConfig } from "hardhat/config";
+import "@nomicfoundation/hardhat-toolbox";
+
+
+const config: HardhatUserConfig = {
+  solidity: "0.8.9",
+  defaultNetwork: "zen",
+  networks: {
+    zen: {
+      url: ‘https://yuma-testnet.horizenlabs.io/ethv1',
+      accounts: {mnemonic: "word1 … word12"},
+      gasPrice: "auto"
+    }, 
+  
+  }
+};
+
+
+export default config;
+```
+
+### Compile a Contract
+In your project folder, create a folder called, **contracts/** to store all your smart contracts. Use the following sample contract files to compile and test. 
+
+#### Sample contract ERC20.sol
+
+Use the [ERC20.sol](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.0.0/contracts/token/ERC20/ERC20.sol) file in GitHub. Import this file to your Hardhat contracts folder by using the script:
+
+
+/ SPDX-License-Identifier: MIT pragma solidity ^0.8.0;
```

Comment:
This doesn't seem to be formatted correctly on the website. I assume this is supposed to be in a code block?